Packs/Core/Playbooks/playbook-Suspicious_process_execution_by_scheduled_task_on_a_sensitive_server_README.md

```diff
@@ -0,0 +1,56 @@
+This playbook handles "Suspicious process execution by scheduled task on a sensitive server" alerts.
+
+Playbook Stages:
+
+Investigation:
+
+During the alert investigation, the playbook will perform the following:
+
+- Checks the suspicious process reputation.
+- Searches for Cortex XSIAM agent alerts related to any malicious activity on the server.
+
+Remediation:
+
+- Remediation actions will be taken if the suspicious process reputation is malicious, or if a related alert is found. In such cases, the playbook will disable the scheduled task, terminate the malicious process, and close the alert.
```

Comment:
for the different remediation actions, change it to a list of actions

Reply:
Done.